wasp/apps/flashlight.py

```diff
@@ -21,19 +20,31 @@ class TorchApp(object):
 
     def foreground(self):
         """Activate the application."""
-        self.draw()
         wasp.system.request_tick(1000)
+        wasp.system.request_event(wasp.EventMask.TOUCH)
 
         self._brightness = wasp.system.brightness
         wasp.system.brightness = 3
+        self.n_touch = 0
```

Comment:
private attributes should start with `_` (like `_brightness`). besides, shouldn't this line go in `__init__`?

Reply:
thanks for the "_". There is currently no __init__ implemented so I didn't know if it was worth adding it as I'm not familiar with how to optimize compiled code. Do you think I should add it? The __init__ method would be almost empty.

Reply:
> Do you think I should add it?

If the behaviour we want is

> to have the flashlight always start at the same point

then you're right in putting it here in foreground()